assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -467,15 +468,23 @@ export function registerDefaults( widgetsAPI ) {
 		widgetsAPI.registerWidget(
 			'keyMetricsEventDetectionCalloutNotification',
 			{
-				Component: WidgetNull,
+				Component: ConversionReportingNotificationCTAWidget,
 				width: [ widgetsAPI.WIDGET_WIDTHS.FULL ],
 				priority: 0,
 				modules: [ 'analytics-4' ],
-				isActive: () => {
+				isActive: ( select ) => {
 					if ( ! isFeatureEnabled( 'conversionReporting' ) ) {
 						return false;
 					}
 
+					if (
+						! select(
+							MODULES_ANALYTICS_4
+						).hasNewConversionReportingEvents()
```

Comment:
Before calling this selector we need to ensure that the Analytics module is connected.

Reply:
Is that necessary? This widget is hooked into key metrics widget area, the widget context of that area will not be active until KM widget area is shown - which is shown once GA4 module is connected, so we should be pretty safe this selector won;t be invoked otherwise